scholarx/2022/index.html

```diff
@@ -635,6 +635,7 @@ <h1 class="mb-0 text-onelive">Take a look at our past</h1>
 <script src="/assets/js/navigation.js"></script>
 <!-- External JS file for the JS functions-->
 <script src="./functions.js"></script>
+<script src="/assets/js/functions.js"></script>
 
```

Comment:
This is also an unwanted line right? Remove it @gajindubandara 

```suggestion
```

Reply:
I didn't add that line; it has been that way from the beginning. It was probably used to clearly differentiate between the above code section and the below code section.

Reply:
Ok then